test/16_supervisor_releases.ts

```diff
@@ -126,18 +127,19 @@ export default () => {
 								},
 							})
 							.expect(200);
+						assertExists(deviceInfo);
 						const { body: nativeSupervisorRelease } = await pineUser
 							.get({
 								resource: 'release',
-								id: deviceInfo.should_be_managed_by__release.__id,
+								id: deviceInfo!.should_be_managed_by__release.__id,
```

Comment:
I think this `!` is not necessary now that you have called `assertExists` on deviceInfo (same valid for all other places)

Reply:
Fixed